projects/plugins/jetpack/tests/php/modules/shortcodes/test-class.vimeo.php

```diff
@@ -186,7 +186,7 @@ public function test_replace_url_with_iframe_in_the_content( $url, $video_id ) {
 
 		global $post;
 
-		$post = self::factory()->post->create_and_get( array( 'post_content' => $url ) );
+		$post = self::factory()->post->create_and_get( array( 'post_content' => "[vimeo $url]" ) );
```

Comment:
Since the content goes from just a URL to a shortcode with a URL, should the function name be updated accordingly?

Reply:
Renamed to `test_shortcodes_vimeo_replace_url_with_iframe_in_the_content`